lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -1884,6 +1906,46 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
     visitPart(RST->getReferentType(), optionalKind);
   }
   
+  Optional<ValueDecl *> findRenamedDecl(const swift::AvailableAttr *AvAttr, const swift::Decl *D) {
+    if (!isa<ValueDecl>(D))
+      return None;
+    
+    SmallVector<ValueDecl *, 4> lookupResults;
+    auto renamedParsedDeclName = swift::parseDeclName(AvAttr->Rename);
+    auto renamedDeclName = renamedParsedDeclName.formDeclName(D->getASTContext());
+    
+    auto declContext = D->getDeclContext();
+    Optional<ValueDecl *>renamedFuncDecl = None;
+
+    if (isa<ClassDecl>(D) || isa<ProtocolDecl>(D)) {
+      UnqualifiedLookup lookup(renamedDeclName.getBaseIdentifier(), declContext->getModuleScopeContext(), nullptr);
```

Comment:
Nitpick: You can pass `UnqualifiedLookup::Flags::TypeLookup` to the "options" parameter here to make the lookup slightly more efficient.